persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -1,30 +1,53 @@
-CREATE TABLE IF NOT EXISTS member (
-  id VARCHAR(36) NOT NULL PRIMARY KEY,
-  email VARCHAR(36),
-  first_name VARCHAR(36),
-  last_name VARCHAR(36),
+CREATE TABLE
```

Comment:
As a general rule you should never change migration SQL files, instead you should add a second sql file (V2__XXX.SQL) that builds on the existing SQL file. This ensures that other user's flyway will recognize the difference and update their database. In this case it may be fine since you're changing pretty much everything but going forward be careful with this.